### PR TITLE
Updates to Zendesk destination following PM comments

### PR DESCRIPTION
**Pull request by mohan-h**
Updates addressing

1. Updated use-case
2. Elaborated the mappings section, minor restructuring and added details for non-mandatory attribute mappings

### CODE REVIEW

File: help/destinations/catalog/crm/zendesk.md [hunk 1]
```diff
@@ -112,21 +112,23 @@ To correctly map your XDM fields to the [!DNL Zendesk] destination fields, follo
...
     ![Platform UI screenshot example with attribute mappings.](../../assets/catalog/crm/zendesk/mappings.png)
 
-        >[!IMPORTANT]
-        >
-        >Both the target field mappings are mandatory and required for [!DNL Zendesk] to work.
-        >
-        >The mapping for *Last Name* or *Name* is required otherwise the [!DNL Zendesk] API does not respond with any error and any attribute value passed is ignored.
+>[!IMPORTANT]
+>
+>Both the target field mappings are mandatory and required for [!DNL Zendesk] to work.
```
Comment (vfalon) [inline help/destinations/catalog/crm/zendesk.md]: Line 129: I am confused. Which "both" target fields are required?

Also, mandatory and required are the same thing. Please rephrase this note by clearly calling out which mappings are required.

**Review by vfalon — CHANGES_REQUESTED** [review]
Thanks for the updates, @mohan-h ! One comment about the mandatory mappings to address. It's not clear to me what the message is there.

File: help/destinations/catalog/crm/zendesk.md (hunk 1 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/zendesk.md]: Thanks, sorry that was missed.

File: help/destinations/catalog/crm/zendesk.md [hunk 2]
```diff
@@ -112,21 +112,21 @@ To correctly map your XDM fields to the [!DNL Zendesk] destination fields, follo
 
 1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
 1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
-1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select custom attributes]** category and select an attribute as needed.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select attributes]** category and select from the displayed schema matching a desired attribute name from the list of [[!DNL Zendesk] Contacts API fields](https://developer.zendesk.com/api-reference/sales-crm/resources/contacts/).
```
Comment (vfalon) [inline help/destinations/catalog/crm/zendesk.md]: Looking at the mapping in the UI, the target schema is already configured for Zendesk. So what is the point of directing readers to the Zendesk API docs? They can just select one of the attributes, they don't need to validate that they match what Zendesk has. They are already validated.

Let's rephrase to:

In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** category and select a target identity, or choose the **[!UICONTROL Select attributes]** category and select one of the supported schema attributes.

File: help/destinations/catalog/crm/zendesk.md [hunk 3]
```diff
@@ -112,21 +112,21 @@ To correctly map your XDM fields to the [!DNL Zendesk] destination fields, follo
...
+
     * Repeat these steps to add the following mappings between your XDM profile schema and your [!DNL Zendesk] instance:
         |Source Field|Target Field| Mandatory|
         |---|---|---|
-        |`xdm: person.name.lastName`|`Attribute: last_name` <br>or `Attribute: name`| Yes |
+        |`xdm: person.name.lastName`|`xdm: last_name` <br>or `xdm: name`| Yes |
```
Comment (vfalon) [inline help/destinations/catalog/crm/zendesk.md]: Mapping `lastName` to `name` does not work. I can only map `lastName` to `lastName`. Please verify this information, since it doesn't look right.

File: help/destinations/catalog/crm/zendesk.md [hunk 4]
```diff
@@ -112,21 +112,21 @@ To correctly map your XDM fields to the [!DNL Zendesk] destination fields, follo
...
         |`IdentityMap: Email`|`Identity: email`| Yes |
```
Comment (vfalon) [inline help/destinations/catalog/crm/zendesk.md:120]: The UI doesn't require me to map the Email identity namespace. I can go through the flow without it. Why is it marked as mandatory?

**Review by vfalon — CHANGES_REQUESTED** [review]
@mohan-h the mapping information still doesn't seem right. I've checked the UI and mapping lastName to name does not work. I can only map lastName to lastName. Please verify this information, since it doesn't look right.

Also, the UI doesn't require me to map the `Email` identity namespace. I can go through the flow without it. Why is it marked as mandatory?

File: help/destinations/catalog/crm/zendesk.md [hunk 5]
```diff
@@ -112,21 +112,20 @@ To correctly map your XDM fields to the [!DNL Zendesk] destination fields, follo
 
 1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
 1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
-1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select custom attributes]** category and select an attribute as needed.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** category and select a target identity, or choose the **[!UICONTROL Select attributes]** category and select one of the supported schema attributes.
     * Repeat these steps to add the following mappings between your XDM profile schema and your [!DNL Zendesk] instance:
```
Comment (vfalon) [inline help/destinations/catalog/crm/zendesk.md]: Repeat these steps to add the following mandatory mappings[...]

File: help/destinations/catalog/crm/zendesk.md [hunk 6]
```diff
@@ -112,21 +112,20 @@ To correctly map your XDM fields to the [!DNL Zendesk] destination fields, follo
...
 
     * An example using these mappings is shown below:
     ![Platform UI screenshot example with attribute mappings.](../../assets/catalog/crm/zendesk/mappings.png)
 
-        >[!IMPORTANT]
-        >
-        >Both the target field mappings are mandatory and required for [!DNL Zendesk] to work.
-        >
-        >The mapping for *Last Name* or *Name* is required otherwise the [!DNL Zendesk] API does not respond with any error and any attribute value passed is ignored.
+>[!IMPORTANT]
```
Comment (vfalon) [inline help/destinations/catalog/crm/zendesk.md:126]: This note needs rephrasing. Let's use this:

The `Attribute: last_name` and `Identity: email` target mappings are mandatory for this destination. If these mappings are missing, any other mappings are ignored and not sent to [!DNL Zendesk].

File: help/destinations/catalog/crm/zendesk.md [hunk 7]
```diff
@@ -7,13 +7,13 @@ last-substantial-update: 2023-03-14
 
 [[!DNL Zendesk]](https://www.zendesk.com) is a customer service solution and sales tool.
 
-This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Zendesk] Contacts API](https://developer.zendesk.com/api-reference/sales-crm/resources/contacts/), to create and update identities within a segment as contacts within [!DNL Zendesk].
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Zendesk] Contacts API](https://developer.zendesk.com/api-reference/sales-crm/resources/contacts/), to **create and update identities** within a segment as contacts within [!DNL Zendesk].
```
Comment (vfalon) [inline help/destinations/catalog/crm/zendesk.md:10]: Any reason for bolding "create and update identities"? I think it's clear enough what that does even without bolding it.

**Review by vfalon — CHANGES_REQUESTED** [review]
@mohan-h thanks for making these updates. See my comments below.

File: help/destinations/catalog/crm/zendesk.md (hunk 7 above)
Comment (vgiurgiu) [reply help/destinations/catalog/crm/zendesk.md:10]: I can provide context here: for another connector (I forget which one), it wasn't clear from the description what the connector does, so I suggested that we bold the section where we say that the connector creates and updates identities. 

@mohan-h - I see that you have taken the same approach here. I think that's fine and I don't see any harm in bolding this part.

File: help/destinations/catalog/crm/zendesk.md (hunk 5 above)
Comment (mohan-h) [reply help/destinations/catalog/crm/zendesk.md]: Rewording to _Repeat these steps to add the following mandatory mappings, you can also add any other attributes you want to update between your XDM profile schema and your [!DNL Zendesk] instance_

File: help/destinations/catalog/crm/zendesk.md [hunk 8]
```diff
@@ -164,4 +163,19 @@ All [!DNL Adobe Experience Platform] destinations are compliant with data usage
...
\ No newline at end of file
+* [Custom Fields](https://developer.zendesk.com/api-reference/sales-crm/requests/#custom-fields)
+
+### Changelog
+
+This section captures the functionality and significant documentation updates made to this destination connector.
+
++++ View changelog
+
+|Release month|Update type|Description|
```
Comment (vfalon) [inline help/destinations/catalog/crm/zendesk.md:174]: The changelog section needs to provide clearer information for a reader. Try to read the section from a customer point of view, and see if you understand what the actual change was. There are also some typos in this section.

Let's rephrase it to something like this:

|Release month|Update type|Description|
|---|---|---|
|April 2023|Documentation update| <ul><li>We updated the [use-cases](#use-cases) section with a clearer example of when customers would benefit from using this destination.</li><li>We updated the [mapping](#mapping-considerations-example) section to reflect the correct required mappings. The `Attribute: last_name` and `Identity: email` target mappings are mandatory for this destination. If these mappings are missing, any other mappings are ignored and not sent to [!DNL Zendesk].</li> <li>We updated the [mapping](#mapping-considerations-example) section with clear examples of both mandatory and optional mappings.</li></ul> |
|March 2023|Initial release|Initial destination release and documentation publish.|
